### PR TITLE
Here are the changes I thought I'd made to clean up use case reduplication

### PR DESCRIPTION
**Pull request by bumblefudge**
They were sitting in a branch unpulled.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -248,66 +248,9 @@ <h3>
         </h3>
 
         <p class="issue">
-These should be in a USE-CASES document.
+There should be a link here to the distinct USE-CASES document.
```
Comment (msporny) [inline index.html:251]: Let's just point to the Google Docs file for now. Add a link pointing there, please.

File: use_cases.md [hunk 2]
```diff
@@ -100,53 +100,6 @@ is completely hidden from a service provider.
 
 Would it make sense to try and distill a Focal Use Case from Adrian Gropper's [Covid Use Case document](https://github.com/agropper/secure-data-store/blob/master/COVID-19_Health_Report_Use_Case.md)?  I'm happy to take a crack at the distillation into a first draft if I get confirmation that it's not wasted effort. I think it might be productive to double-check that the functional requirements 1.3 & 1.9, as well as the technical requirements of 3.2, make sense across a change of SDS provider.
 
-# ​X.​ Deployment topologies
```
Comment (msporny) [inline use_cases.md:103]: My assumption is that you are moving these to the use cases / requirements document as well?

**Comment by bumblefudge** [comment]
Sorry again to both @OR13 and @msporny, I'll get the hang of all this yet!

File: index.html [hunk 3]
```diff
@@ -317,26 +260,31 @@ <h4>
...
+            <li>
+<strong>Self-Hosted and/or Home-based Server:</strong> Alice wants to host her own SDS software instance, on her own server.
+            </li>
+            <li>
+<strong>Support Low Power Devices/Non-private computing:</strong> To support users without access to private computing resources, the following three components need to be considered:
+* Secure Storage
+* Key vault - private key storage and recovery (Key management)
+* Trusted computing - computational resources which have access to private keys and plain text private data
+  </li>
           </ul>
```
Comment (msporny) [inline index.html:288]: This whole section has been reformatted to not break at 78 characters, please put it back to the way it was. Putting stuff on single lines makes it difficult for the Editors of the document who don't have line wrapping enabled (some of us turn this off by default to make sure preformatted text isn't wonky), and it makes putting in small changes more difficult, when we quote text for fixes in PRs, we have to scan massive paragraphs.

In other words, if you're in ReSpec, break lines before the 78 character column mark, please.

If you don't know how to do this, @bumblefudge, I can fix it after merging, but I'd rather you do it since you're going to be doing a lot of editing (which I'm very thankful for!). :)

Comment (bumblefudge) [reply index.html:288]: I do not *currently* know how to do this and have been editing everything in github.com, but happy to install ReSpec over the weekend and cleave the 78char line limit for all future pulls! Maybe in the interest of time I could ask you to do it just this once, pretty pretty please?

Comment (dmitrizagidulin) [reply index.html:288]: @bumblefudge @msporny @OR13 - since this is a PR from a fork, I couldn't edit it directly, so the requested whitespace fixes are over at https://github.com/decentralized-identity/secure-data-store/pull/84 (which supercedes this one).

**Comment by OR13** [comment]
Suggest closing this if its too stale / has been addressed by subsequent PRs.

File: index.html (hunk 3 above)
Comment (bumblefudge) [reply index.html:288]: Just to be explicit, Dmitri's changes are fine with me and seem to address the requested change! Will figure out hard wraps before next PR :D
